README.md

```diff
@@ -34,6 +34,8 @@ applications beside manual checks:
 
         curl -H "Accept: application/json" https://security.symfony.com/check_lock -F lock=@/path/to/composer.lock
 
+It is convenient to include the call to the Symfony Web Service in any build plans or existing Continuous Integration workflow, so that you can check for any new unaddressed vulnerabilities every time your application is built. This can be achieved by including the curl command (from above) in any build.sh (or equivalent) file (pointing to the local composer.lock). 
```

Comment:
I would press this more urgent and say something along the lines of 
"Its advised to include..."

Reply:
And maybe refer to https://github.com/sensiolabs/security-checker as well, as an alternative to a barebones curl.

Reply:
Thanks for your feedback @BackendTea. I'll update the Pull Request taking these comments into account.

Reply:
Hi @BackEndTea , I have updated the Pull Request. Hope you can review these updates whenever you have a chance to do so.